components/eamxx/src/mct_coupling/atm_comp_mct.F90

```diff
@@ -149,7 +150,8 @@ subroutine atm_init_mct( EClock, cdata, x2a, a2x, NLFilename )
          dom=dom_atm, &
          infodata=infodata)
     call seq_infodata_getData(infodata, atm_phase=phase, start_type=run_type, &
-                              username=username, case_name=caseid, hostname=hostname)
+                              username=username, case_name=caseid, &
+                              rest_case_name=rest_caseid, hostname=hostname)
```

Comment:
@rljacob Is `rest_case_name` always defined, even if this is a "regular" restart (not branch)? I assumed yes. That is, for a simple CONTINUE_RUN, I expect rest_case_name==case_name...

Reply:
Yes its read from the restart file.